src/affinder/affinder.py

```diff
@@ -136,6 +144,7 @@ def start_affinder(
         moving_points: Optional['napari.layers.Points'] = None,
         model: AffineTransformChoices,
         output: Optional[pathlib.Path] = None,
+        keep_pts: bool = False,
```

Comment:
I would reverse this, call it `delete_points_when_done`. Label: "Delete points layers when done." Tooltip: "If ticked, the points layers used in alignment will be deleted when clicking 'Finish'."